api/routes/api.php

```diff
@@ -39,12 +39,13 @@
     'namespace' => 'Api\\'
 ], function () {
     Route::group([
-        'prefix' => '/event-types',
+        'prefix' => '/eventTypes',
```

Comment:
Usually, it is not a good practice to change the route to the existing endpoints, because it can break a lot (including Postman endpoints that devs should already have locally).